openapi/components/schemas/ReportMonthlyRecurringRevenue.yaml

```
@@ -0,0 +1,53 @@
+type: object
+properties:
+  data:
+    type: array
+    items:
+      type: object
+      properties:
+        period:
+          description: Revenue month.
+          type: string
+          pattern: '^\d{4}-\d{2}$'
+        total:
+          description: The total MRR amount.
```

nitpick: It's good to describe the acronym in its first instance. 

```suggestion
          description: Total Maintenance Margin Requirement (MMR) amount.
```

---

Agreed, but it's MRR, not MMR, so
```suggestion
          description: Total Monthly Recurring Revenue (MRR) amount.
```